Makefile

```diff
@@ -14,8 +14,8 @@ shell:
 	docker exec -it feature_backend /bin/bash
 
 block:
-	docker exec -it feature_backend /bin/bash -c 'echo "127.0.0.1\tapi.feature-service.com" >> /etc/hosts'
+	docker exec -it feature_backend /bin/bash -c 'echo "127.0.0.1 cdn.growthbook.io" >> /etc/hosts'
```

Comment:
`make block` doesn't work because Growthbook saves connection in a pool when the app starts and reuses it.